drivers/counter/Kconfig.ace

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2022 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+config ACE_V1X_ART_COUNTER
+	bool "DSP ART Wall Clock for ACE V1X"
+	depends on SOC_SERIES_INTEL_ACE1X && DT_HAS_INTEL_ACE_ART_COUNTER_ENABLED
```

Comment:
why the double dependency? The idea with DT_HAS_... macros is to replace the SOC dependency, of an soc does not define this in DTS, it will not be enabled.